### DISCUSSION TITLE
How should we engage with the community and also users

### DISCUSSION THREAD

**Discussion by fforootd** [zitadel/zitadel#1919] [Contributors]
## How can we improve the community engagement

- Discuss idea, new features, decisions on public
- Engage with users and contributors openly 

## What tools should we choose

- GH Discussion for chat style exchanges
- GH Issues for features, bugs after a discussion

## Further ideas

- Plan Virtual get togethers
- Plan Physical get togethers

Anymore Ideas?

**Comment by adlerhurst** [comment]
I'm not sure if it's possible to form a community with discussions. Discussions is much more formal for me than a chat tool. I think some communities use gitter which could be a good tool for this use case.

Edit: just saw that we won't provide a chat for the community in another discussion. so ignore this one

**Comment by fforootd** [comment]
Why do you think it is not suitable?

I mean GH discussions are topic oriented and we can even use general topics to discuss random things.

Regarding Gitter, this project is in a strange state, since it left gitlab and joined matrix.
However we could give it a spin.

I just think Slack is a bad choice because it is closed and not really searchable.

**Reply by adlerhurst** [reply]
I would start a discussion on github if it's technically and product related. If I want to start a challenge "who is able to design the ugliest login page" i won't start a gh discussion as it wont bring any benefits. I'm not sure if that's just me or not. 

I think the most projects use slack, discord or gitter. I think i haven't seen many others. At the moment i have no opinion which fits best.

**Reply by fforootd** [reply]
Ok but what benefit would a chat bring there?

I mean challenges could be done with discussions, with a separate category.

At the moment I just don’t get what a chat would serve for a purpose :D
Because in the end all our discussions and decision should be made public, and that’s where a chat fails.

**Reply by adlerhurst** [reply]
i commented here before i saw https://github.com/caos/zitadel/discussions/1919 which answered my comment. Thanks for your expainations.

I just wanted to point out, that I write texts differently when I'm using a chat than on gh discussions. Personally I wouldl not use gh discussions for off topic chats. But maybe thats just my fault.

**Reply by fforootd** [reply]
Well ok we could establish a chat for fooling around 😁 that does not hurt.

**Comment by fforootd** [comment]
FYI we are currently testing gitter. https://gitter.im/caos/zitadel

It looks promising as matrix tries to integrate it into their ecosystem. https://matrix.org/blog/2020/09/30/welcoming-gitter-to-matrix
